src/vignette/http/routes.clj

```diff
@@ -76,6 +76,17 @@
                   :thumbnail-mode "scale-to-width"
                   :width size-regex}))
 
+(def scale-only-down-route
```

Comment:
Looks like this route isn't getting used anywhere, it needs to be added to [`app-routes`](https://github.com/Wikia/vignette/blob/platform-1236-implement-scale-to-width-down/src/vignette/http/routes.clj#L107)